dagql/idtui/frontend.go

```diff
@@ -101,10 +103,13 @@ func (r renderer) renderIDBase(out *termenv.Output, call *callpbv1.Call) error {
 		parent = parent.Foreground(moduleColor)
 	}
 	fmt.Fprint(out, parent.String())
+	if r.Verbosity > 0 && call.ReceiverDigest != "" {
+		fmt.Fprint(out, out.String(fmt.Sprintf("@%s", call.ReceiverDigest)).Foreground(faintColor))
+	}
```

Comment:
Hm this adds a bit of noise for me.

I use `-v` by itself just to get the pretty logs to stick around (I use terminal scrollback a lot if you can't tell :laughing:) - I'm actually starting to wonder if we should split persistence out from `-v`.

Reply:
Ah this was mostly me adding it because I needed it in the moment, I can bump the verbosity level to `-vv`. I think `-v` being mostly just persistence is ideal.

There are a bunch of individual flags we could probably add but I'd rather try to just stick with the verbosity level mindset as long as possible and try to curate it, since any other flag I can think of tends to be long, and I'm not gonna want to type it, and if we give these flags short names that's potentially claiming a lot of short-flag-name real estate.